lambdas/static-asset-uploader/index.js

```diff
@@ -209,6 +209,14 @@ exports.handler = (event, context) => {
                                 ACL: "public-read"
                             },
                             options = {}
+
+                            let suffix = event.ResourceProperties.MarketplaceSuffix
```

Comment:
Is it guaranteed that event.ResourceProperties.MarketplaceSuffix will always be defined? Or is it possible that this could throw a ReferenceError?

Reply:
Yep! `event` is the lambda invocation event, and `ResourceProperties` is set on the invocation event by CloudFormation. If the staticAssetUploader is invoked by something other than CFN, `ResourceProperties` could be undefined, but right now, staticAssetUploader is intended to only be used from CFN.